src/components/Form/Picklist.js

```diff
@@ -67,10 +75,10 @@ export class PicklistRaw extends Component {
   };
 
   renderPicklistItems() {
-    const { onSelect, items } = this.props;
+    const { items } = this.props;
 
     return items.map((item) => {
-      const boundClick = () => onSelect(item.key);
+      const boundClick = () => this.onSelect(item.key);
```

Comment:
Mighty as well change method `onSelect` to `getSelectHandler(itemKey)` here as it avoids creating functions in render